src/lib/mina_base/party.ml

```diff
@@ -795,7 +795,7 @@ module Body = struct
           ; sequence_events : Events'.Stable.V1.t
           ; protocol_state_precondition :
```

Comment:
change this to `network_precondition` to be consistent?

Reply:
I did this via the `@name` annotation for now because I didn't want this PR to balloon in scope. Was planning on making an issue for addressing that later. Is that okay?